### PR TITLE
Add pages on Zapier integration

### PR DESCRIPTION
**Pull request by svix-nolan**

### CODE REVIEW

File: docs/integrations/zapier.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+:::note Upgrades & Modifications
+If you're upgrading the integration after making customizations or adding new event types, you may need to reconcile differences between the last version deployed and the latest version downloaded. We recommend using a version control system like Git to manage the history.
+:::
+
+### Download the auto-generated integration package
+
+You can download an auto-generated integration package from the [integrations page](https://dashboard.svix.com/integrations) on the dashboard shown below.
+
+The download form requires a project name which corresponds to the NPM project name, and a version number which corresponds to both the NPM projection version and the Zapier integration version.
```
[the remark this answers is not in the document]
Comment (svix-liam) [reply docs/integrations/zapier.mdx:53]: ❓  so the Zapier integration version needs to match my npm package version?

File: docs/integrations/zapier.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+# Link an existing Zapier integration to upgrade it
+zapier link
+```
+
+Build & deploy the integration:
+```
+zapier push
+```
+
+Finally, try it out! From the account where you deployed the integration, you can create a Zap and test the integration. It won't be generally available on Zapier until it's released.
```
Comment (svix-liam) [inline docs/integrations/zapier.mdx:80]: 💡  One idea would be to include either a screenshot of a basic Zap here or we could eventually add a separate tutorial on Svix zaps and link to that.

**Review by svix-liam — COMMENTED** [review]
Looks great! I left a few small comments and clarifying questions but this was super clear and thorough IMO.

---

Reminder not to merge until we ship the integrations page

File: docs/integrations/zapier.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (svix-nolan) [reply docs/integrations/zapier.mdx:53]: Yes, but the cause and effect the other way around. So when you want to change the Zapier integration version, you would change the NPM package version. Will clarify that a bit

File: docs/integrations/zapier.mdx (hunk 2 above)
Comment (svix-nolan) [reply docs/integrations/zapier.mdx:80]: Good call! Details on the zap UX is missing.

Also need to add some details on the auth flow for a zap.

File: docs/integrations/zapier.mdx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+```
+
+Build & deploy the integration:
+```
+zapier push
+```
+
+### Try it out
+
+Finally, try it out! From the account where you deployed the integration, you can create a Zap and test the integration. It won't be publically available on Zapier until it's released, but you can share the integration with your users.
```
Comment (svix-liam) [inline docs/integrations/zapier.mdx:86]:
```suggestion
Finally, try it out! From the account where you deployed the integration, you can create a Zap and test the integration. It won't be publicly available on Zapier until it's released, but you can share the integration with your users.
```

File: docs/integrations/zapier.mdx [hunk 4]
```diff
@@ -83,7 +83,7 @@ zapier push
 
 ### Try it out
 
-Finally, try it out! From the account where you deployed the integration, you can create a Zap and test the integration. It won't be publically available on Zapier until it's released, but you can share the integration with your users.
+Finally, try it out! From the account where you deployed the integration, you can create a Zap and test the integration. It will not be publicly available on the Zapier website until released, but you can share the integration with your users.
```
Comment (svix-liam) [inline docs/integrations/zapier.mdx:86]: I think it might have sounded a bit better as `until it's released` but up to you.

**Review by svix-liam — COMMENTED** [review]
LGTM

**Review by svix-liam — APPROVED** [review]

File: docs/integrations/advanced-zapier.mdx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+To provide a better user experience that hides the Svix constructs, we recommend using [OAuth2](https://platform.zapier.com/cli_docs/docs#oauth2) or [Session](https://platform.zapier.com/cli_docs/docs#session) authentication. See the [Zapier Platform authentication](https://platform.zapier.com/cli_docs/docs#authentication) docs for more information.
+- With OAuth2, Zapier will redirect your user to your site where you can authenticate them and send Zapier back an access token.
+- With Session Auth, Zapier will show a login form for your user to provide their username and password. Those are securely sent to your service where you'll send Zapier back a session token.
+
+With either authentication flow, you can make another request to your APIs to exchange the access/session token for the Svix application ID and integration key.
+An example of that API endpoint for a Flask-based service might look like the following:
+
+```
+TODO
```
Comment (svix-nolan) [inline docs/integrations/advanced-zapier.mdx:50]: Whoops

File: docs/integrations/zapier.mdx [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+```
+
+### That's it!
+
+![congratulations](../img/congratulations.png)
+
+You're done! You have a working Zapier integration for your Svix-powered webhooks. :)
+
+Try it out! The next section walks through the experience of creating a Zap with a webhook trigger.
+Then read on to see how you can customize your integration and release it publicly on the Zapier website.
```
Comment (tasn) [inline docs/integrations/zapier.mdx:206]:
```suggestion
You're done! You have a working Zapier integration for your Svix-powered webhooks. :)

Try it out! The next section walks through the experience of creating a Zap with a webhook trigger.
Then read on to see how you can customize your integration and release it publicly on the Zapier website.

![congratulations](../img/congratulations.png)
```

**Review by tasn — APPROVED** [review]

**Pull request merged**